tests/ui/pages/catalog/catalog_page.ts

```diff
@@ -0,0 +1,59 @@
+import Button from '../../elements/button';
+import BasePage from '../base-page';
+
+const SELECTORS = {
+  clearAll: `button:has-text('Clear All')`,
+  filterWithSelect: filterName => `.MuiGrid-root.css-19dbjmo:has-text('${filterName}')`,
+  filterWithInput: filterName =>
+    `.MuiGrid-root.MuiGrid-item.MuiGrid-grid-xs-12.css-1vkoyw5:has-text('${filterName}')`,
+  filterSelect: `[role="button"]`,
+  filterInput: `[type="text"]`,
+  searchBar: `[placeholder="Search"]`,
+  searchButton: `.sc-kDDrLX.gpLWbw > div > div > div > div > [type="button"]`,
```

Comment:
Pls remove the unused selector
p.s. in this case ill better use '.sc-kDDrLX.gpLWbw button[type="button"]' as selector